neqo-crypto/src/aead.rs

```diff
@@ -59,7 +59,8 @@ impl Aead {
         unsafe { Self::from_raw(version, cipher, s, prefix) }
     }
 
-    pub fn expansion(&self) -> usize {
+    #[must_use]
+    pub fn expansion() -> usize {
```

Comment:
Removing `&self` here might be something that clippy wants, but I'm not sure that I do.

Reply:
Yeah I'm not really sure about that either 😁